test/dpg/version-tolerant/Expected/AcceptanceTests/DPGCustomizationCustomizedVersionTolerant/dpgcustomizationcustomizedversiontolerant/_operations/_patch.py

```diff
@@ -2,32 +2,34 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 # ------------------------------------
-from typing import Any, Iterable
+from typing import Any, Iterable, Union, cast
 from azure.core.polling import LROPoller
 from ..models import *  # pylint: disable=wildcard-import,unused-wildcard-import
-from ._operations import DPGClientOperationsMixin as DPGClientOperationsMixinGenerated
+from ._operations import DPGClientOperationsMixin as DPGClientOperationsMixinGenerated, JSONObject
 
 
 class DPGClientOperationsMixin(DPGClientOperationsMixinGenerated):
     def get_model(self, mode: str, **kwargs: Any) -> Product:
         response = super().get_model(mode, **kwargs)
         return Product(**response)
 
-    def post_model(self, mode: str, input: Product, **kwargs: Any) -> Product:
+    def post_model(self, mode: str, input: Union[Product, JSONObject], **kwargs: Any) -> Product:
```

Comment:
Here is trying to make parameter as generic as the method in parent class.

Or shall we change type of input as `JSONObject` here? Since the user can still provide a `Product` to it.